gleam.toml

```diff
@@ -11,13 +11,13 @@ gleam = ">= 0.32.0"
 # links = [{ title = "Website", href = "https://gleam.run" }]
 
 [dependencies]
-gleam_stdlib = "~> 0.31"
-gleam_http = "~> 3.5"
-gleam_erlang = "~> 0.22"
-gleam_crypto = "~> 1.0"
-mist = "~> 0.14"
-sqlight = "~> 0.8"
-wisp = "~> 0.5"
+gleam_stdlib = "~> 0.38"
+gleam_http = "~> 3.6"
+gleam_erlang = "~> 0.25"
+gleam_crypto = "~> 1.3"
+mist = "~> 1.2"
+sqlight = "~> 0.9"
+wisp = "~> 0.15"
```

Comment:
This is making the constraints tighter, which is not the same thing as upgrading the versions used. To upgrade the versions you can run `gleam update`.
Mist here is the only one that benefits from changing the constraints as it has had v1 released, which is outside of `~> 0.14`.

Could you update these constraints to use the currently recommended syntax rather than `~>` please 🙏

Reply:
Happily! But what is the recommended syntax? something like `>= 2.0.0 and < 2.1.0` (looking at https://hexdocs.pm/elixir/Version.html#module-requirements)

Reply:
Happily! But what is the recommended syntax? something like >= 2.0.0 and < 2.1.0 (looking at https://hexdocs.pm/elixir/Version.html#module-requirements)